### PR TITLE
Completed all 3 Labs

### PR DESCRIPTION
**Pull request by fasteiner**

### CODE REVIEW

File: src/Lab.3.Bouncer/Lab.3.Bouncer.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    [Parameter(ValueFromPipeline = true, Mandatory = true, Position = 0)]
+    public string name { get; set; } = "";
+    protected override void ProcessRecord()
+    {
+        Person person = new Person(name, Lorem.DateTime(1900));
+        WriteObject(person);
+    }
+}
+
+[Cmdlet("Assert", $"{Consts.ModulePrefix}Age")]
```
Comment (PalmEmanuel) [inline src/Lab.3.Bouncer/Lab.3.Bouncer.cs]: Just like with VerbsCommon.Get on line 5, you can use VerbsLifecycle.Assert to reference the string "Assert" from the approved verbs. It's the same result though, just a small tip :)

**Comment by PalmEmanuel** [comment]
Well done with the labs! All tests are passing except one in lab 3 👍😎

**Comment by fasteiner** [comment]
Interesting, that the tests fail in the repo, but run on my local machine:
image

Any ideas what might be the issue?

**Comment by fasteiner** [comment]
Ahh I see launched the wrong Pester file, that's why it ran through :(

**Comment by PalmEmanuel** [comment]
That makes sense, well done! 🥳

**Comment by JustinGrote** [comment]
🎉🎉🎉

**Pull request closed without merging**